src/services/functionAppService.test.ts

```diff
@@ -248,10 +283,19 @@ describe("Function App Service", () => {
     }, slsService["artifact"])
   });
 
-  it("throws an error with no zip file", async () => {
+  it("uses default name when no artifact provided", async () => {
```

Comment:
previous tests was testing a different things, why is it being replace?

Reply:
This test was wrong. It was throwing an error, but for the wrong reason. Now that `getFunctionZipFile` is public, I can test it directly, which is why I renamed the test.